wellcomecollection/catalogue-pipeline
# Index concept source IDs for filtering

## What does this change?

https://github.com/wellcomecollection/platform/issues/5820

Follow-up on [this PR](https://github.com/wellcomecollection/catalogue-pipeline/pull/2746).

Indexes concept source IDs (i.e. LoC IDs, MeSH IDs, or label-derived IDs) in works and images indexes for filtering. 

## How to test

See [this PR](https://github.com/wellcomecollection/catalogue-pipeline/pull/2746).

## How can we measure success?

See [this PR](https://github.com/wellcomecollection/catalogue-pipeline/pull/2746).

## Have we considered potential risks?

See [this PR](https://github.com/wellcomecollection/catalogue-pipeline/pull/2746).
